### DISCUSSION TITLE
How to combine artists that aren't case sensisitve?

### DISCUSSION THREAD

**Discussion by TheMarstonConnell** [gerbera/gerbera#3059] [Q&A]
Gerbera is creating two artists since the `of` in the band name is sometimes capitalized, how do I stop this from happening? Thanks

Example:

<img width="193" alt="image" src="https://github.com/gerbera/gerbera/assets/34043723/a3c0652e-45cf-4243-bbff-52ed36d7a0d8">

**Comment by KarlStraussberger** [comment]
In principle it works if you change all the "wrong" properties. The better solution is to provide a tiny config option that allows to set the behaviour (like `<import case-sensitive-tags="no" ... >`) and you don't have to worry about.

Will have to test that idea but does not look too big.
